### DISCUSSION TITLE
FrePPLe execution log UOM

### DISCUSSION THREAD

**Discussion by gibhsu** [frePPLe/frepple#464] [Q&A]
We started to do our first attempt in our K8 environment for POC. I have the following logs. May I know the unit of measure for Memory? Bit or Byte?

INFO Start step 120 'Plan Size' at 09:58:29

Size information of frePPLe 6.19.0 (Dec  1 2021)

Clusters: 7323

Memory usage:
Model                 	Count	Memory
-----                 	-----	------
Plan                  	1	347
Location              	29	5698
Customer              	184	33120
Supplier              	1	188
Buffer                	71274	28545408
Setup matrix          	0	0
Setup matrix rules    	0	0
Resource              	0	0
Skill                 	0	0
Resource skill        	0	0
Operation             	739449	352170893
Operation material    	125042	23147252
operation resource    	0	0
Calendar              	5	968
Calendar buckets      	5649	740019
Item                  	83326	97026113
Item suppliers        	2948	660352
Item distributions    	43969	9497304
Demand                	17021	6346803
Constraints           	0	0
OperationPlan         	0	0
OperationPlan material	0	0
OperationPlan resource	0	0
Problem               	34074	3374766
String pool           	3694	265968
Total                 		521549231

INFO Finished 'Plan Size' at 09:58:36

**Comment by jdetaeye** [comment]
It's in bytes.

Note that these metrics are only a rough indication of the memory usage. I have seen situations where the actual process size is more than 30% higher than what is reported in the log file (due to memory used eg by the python interpreter, temporary calculation memory, global objects not included in the counts, etc...).

**Reply by gibhsu** [reply]
Understood. We are setting 64G memory so I want to make sure it is sufficient enough.
